src/pages/HomePage.jsx

```diff
@@ -28,6 +29,35 @@ const Home = () => {
     user: `User ${index + 1}`,
     message: `Conversations and its Description Summary`,
   }));
+  const handleFileInputChange = (event) => {
+    const selectedFile = event.target.files[0];
+
+    // Check if the selected file is not null
```

Comment:
Some of the comments in the code are not required , if you put it for your reference please clean everything before making a pull request.